src/gui/src/CapriceAbout.cpp

```diff
@@ -14,7 +15,7 @@ CapriceAbout::CapriceAbout(const CRect& WindowRect, CWindow* pParent, CFontEngin
 {
   SetModal(true);
 	// Override here: specify position of label ourselves:
-	m_pMessageLabel = new CLabel(CPoint(5, 70), this, "Version 4.3.0");
```

Comment:
This explains why you didn't found it on the first commit. I thought it was VERSION_STRING already !